mimic/test/test_nova.py

```diff
@@ -618,6 +618,68 @@ def test_server_resize(self):
             }
         })
 
+    def test_confirm_and_revert_server_resize(self):
+        """
+        After a server finishes resizing, the size must be confirmed or reverted
+        A confirmation action should make the server ACTIVE and return a 204
+        A revert action should change the flavor and return a 202
+        Attempting to revert or confirm that is not in VERIFY_RESIZE state returns a 409
```

Comment:
Could you add a reference to the nova documentation?  This would also help me, as a lazy reviewer, make sure that this is doing the right thing :).